modules/ocf_mirrors/files/healthcheck

```diff
@@ -19,9 +20,33 @@ import requests
 Mirror = namedtuple('Mirror', ['url', 'updated_at'])
 
 
-def get_updated(mirror_url):
-    """Find the time the host was last updated.
+def update_func(healthcheck):
+    if healthcheck == 'debian':
+        return get_updated_debian
+    elif healthcheck == 'unix_timestamp':
+        return get_updated_ts
+    elif healthcheck == 'datetime':
+        return get_updated_datetime
+    else:
+        raise ValueError('Unsupported type: {}'.format(healthcheck))
+
+
+def get_updated_datetime(mirror_url):
+    """Find the time the host was last synced"""
```

Comment:
I'd make this function description more useful, like show what the format looks like generally on the mirror, maybe what this function then returns (although that's pretty clear anyway from how it works).